docs/rfc/active/000-ml_models/README.md

```diff
@@ -0,0 +1,121 @@
+- Start Date: 08/18/2020
+- RFC PR: https://github.com/linkedin/datahub/pull/1812
+- Implementation PR(s): https://github.com/linkedin/datahub/pull/1721
+
+# Machine Learning Models
+
+## Summary
+
+Adding support for trained machine learning models and features metadata cataloging and enabling search and discovery over them. This is a step towards organizing the essential facts of machine learning models in a structured way leading to responsible democratization of machine
+learning and related artificial intelligence technology. The work is inspired by Google's model card [paper](https://arxiv.org/pdf/1810.03993.pdf).
+
+## Motivation
+
+We need to model ML model metadata for transparent model reporting. Below are some of the reasons why storing machine learning model metadata is important:
+- Drawing boundaries around a model's capabilities and limitations: There is a need to store the conditions under which a model performs best and most consistently and if it has some blind spots. It helps potential users of the models be better informed on which models are best for their specific purposes. Also, it helps minimize usage of machine learning models in contexts for which they are not well suited.
+- Metrics and Limitations: A model’s performance can be measured in countless ways, but we need to catalog the metrics that are most relevant and useful. Similarly there is a need to store a model's potential limitations that are most useful to quantify.
+- Ensure comparability across models in a well-informed way: Modeling metadata of ML models allows us to compare candidate models' results across not only traditional evaluation metrics but also along the axes of ethical, inclusive, and fair
+considerations.
+- Promote reproducibility: Often a model is trained on transformed data, there are some preprocessing steps involved in transforming the data e.g. centering, scaling, dealing with missing values, etc. These transforms should be stored as part of model metadata to ensure reproducibility. 
```

Comment:
Few more points on the Motivation section:
1. Helps in discovering (and promote reuse of) existing ML Models across the organization
2. Identify and tag models which uses sensitive data from a governance standpoint.

Reply:
Added, thanks for the suggestion!